deployment/docker/compose-vt-single.yml

```diff
@@ -0,0 +1,119 @@
+services:
+
+  hotrod:
```

Comment:
It would be nice to add a comment mentioning what's this component and why we need it

Reply:
updated description